Aztec/Classes/TextKit/TextStorage.swift

```diff
@@ -135,7 +149,12 @@ open class TextStorage: NSTextStorage {
     /// - Returns: the preprocessed string.
     ///
     fileprivate func preprocessAttachmentsForInsertion(_ attributedString: NSAttributedString) -> NSAttributedString {
-        assert(attachmentsDelegate != nil)
+        // Ref. #727:
```

Comment:
Use a link rather than issue number (since the project could very well be forked).